ci.sh

```diff
@@ -51,8 +51,9 @@ make cross
 sudo mv out/linux-amd64/crc /usr/local/bin/
 popd
 
+crc config set bundle crc_libvirt_*.crcbundle
```

Comment:
```
This patch allow setup and start command to use the same bundle which created
using the PR and don't use what part of crc Makefile.
```
-> `This patch alloows the setup and start commands to usue the same bundle which was created using the PR instead of using the one hardcoded in crc Makefile`